theme/sass/components/_accordion-component.scss

```diff
@@ -0,0 +1,58 @@
+.accordion-component {
+
+  .accordion {
+    background-color: $accent-color-one;
+    color: $secondary-color;
+    font-weight: bold;
+    cursor: pointer;
+    padding: 18px;
+    text-align: left;
+    border: none;
+    outline: none;
```

Comment:
Remove this please, you shouldn't be removing the outline because its there for accessibility reasons here's some read: http://www.outlinenone.com/ i can explain further in person if you wish.

Reply:
done